faunadb-scala/src/main/scala/faunadb/errors/FaunaException.scala

```diff
@@ -28,7 +28,7 @@ case class UnavailableException(response: Option[QueryErrorResponse], message: S
 /**
  * An exception thrown if FaunaDB cannot evaluate a query.
  */
-class BadRequestException(response: Option[QueryErrorResponse], message: String) extends FaunaException(response, message) {
```

Comment:
Interestingly `UnavailableException` above was already a case class.

Reply:
Mhm, that's curious!